3MI0800331/Task2.cpp

```diff
@@ -0,0 +1,76 @@
+#include <iostream>
+using namespace std;
+
+
+struct orderedPair {
+	int first;
+	int second;
+};
+
+void print(const orderedPair& arg) {
+		cout << arg.first << " " << arg.second;
+};
+
+orderedPair compare(const orderedPair& arg1, const orderedPair& arg2) {
+	orderedPair greatest;
+	if (arg1.first > arg2.first) {
+		greatest.first = arg1.first;
+		greatest.second = arg1.second;
+		
+	}
+	if (arg1.first < arg2.first) {
+		greatest.first = arg2.first;
+		greatest.second = arg2.second;
+	}
+	return greatest;
+}
+
+bool isBigger (const orderedPair& arg1, const orderedPair& arg2) {
+	if (arg1.first > arg2.first) {
```

Comment:
That's not the correct way to do the comparison - if `arg1.first == arg2.first` you need to compare `arg1.second vs arg2.second` :P